spec/controllers/projects_controller_spec.rb

```diff
@@ -193,6 +193,19 @@
               expect(assigns[:story].project).to eq(project)
             end
           end
+
+          describe 'when the user change to another project from another team' do
+            let(:new_team)            { create :team }
+            let(:new_project)         { create :project, users: [user] }
+            let(:new_story)           { create :story, project: new_project, requested_by: user }
+            it 'should accept request when it is from registred team' do
+              new_team.projects << new_project
```

Comment:
Try to follow the  [Four-Phase pattern](https://thoughtbot.com/blog/four-phase-test) and add some spaces between each phase.
```
new_team.projects << new_project
user.teams << new_team

get :show, params: { id: new_project }

expect(session[:current_team_slug]).to eq(new_team.slug)
expect(response).to have_http_status(:ok)
```